src/main/resources/assets/supercircuitmaker/lang/de_de.json

```
@@ -0,0 +1,51 @@
+{
+  "itemGroup.supercircuitmaker": "Super Circuit Maker",
+
+  "block.supercircuitmaker.circuit": "Schaltkreis",
+  "block.supercircuitmaker.inspector": "Inspektor",
+  "item.supercircuitmaker.screwdriver": "Schraubendreher",
+  "item.supercircuitmaker.palette": "Palette und Pinsel",
+  "item.supercircuitmaker.tiny_redstone": "Kleiner Haufen von Redstone Staub",
+  "item.supercircuitmaker.tiny_rgb_redstone": "Kleiner Haufen von buntem Redstone Staub",
+  "item.supercircuitmaker.randomizer": "Glowstone-Erreger",
+  "item.supercircuitmaker.delay": "Quartz-Resonator",
+  "item.supercircuitmaker.pulsar": "Ender-Pulsar",
+  "item.supercircuitmaker.note": "Booper",
+  "item.supercircuitmaker.lamp": "Miniaturlampe",
+  "item.supercircuitmaker.button": "Miniaturknopf",
+  "item.supercircuitmaker.adder": "Addierer",
+  "item.supercircuitmaker.subtractor": "Subtrahierer",
+  "item.supercircuitmaker.multiplier": "Multiplikator",
+  "item.supercircuitmaker.divider": "Teiler",
+  "item.supercircuitmaker.constant": "Konstante",
+  "item.supercircuitmaker.platform": "Support-Plattform",
+  "item.supercircuitmaker.redstone_stick": "Mit Redstone beschichteter Stock",

```

```suggestion
  "item.supercircuitmaker.redstone_stick": "Redstone-beschichteter Stock",
```
Maybe? Shorter and imo less awkward. But it doesn't work with the RGB stick so maybe not, to keep it consistent